package.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
Do you think we need npm to track js dependencies? I don't believe there's too many.
Just wanna avoid ballooning the repo with clutter

Reply:
I'm a strong proponent of maintainability, and I believe being explicit is one way to achieve it. Also, I believe `npm install` gives you either errors or warnings if there is no `package.json`.